### ISSUE TITLE
Magisk 18.2 and 19.0 betas crashing System UI when contact permission is enabled for Google App

### ISSUE THREAD

**Issue by divadkha** [topjohnwu/Magisk#1285]
This is on a stock Pixel 2 XL on Android Pie with March security.
I was trying to get Google Pay to work by updating from Magisk 18.1 to beta.
Initially, after updating to Magisk 18.2 my SystemUI would keep crashing after completing a boot, which would lead to forcing a restart and I would eventually have to roll-back to Magisk 18.1.

Then, I tried updating to 19.0 which worked fine. But I realized that because I deleted the data/data/com.google.android.gms folder, some permissions were reset for the Google app.
As soon as I give the Google app permission to access my contacts (for use with Google Assistant), that's when SystemUI would keep crashing. Once I go into app settings for the Google app and disable permissions for contacts, everything works fine again.

TL;DR - Magisk 18.2/19.0 betas would keep crashing SystemUI if contact permission is enabled for Google app. Works fine when contact permission is disabled for Google app.

**Comment by topjohnwu** [comment]
No logs
